core/server/proxy/src/main/java/alluxio/web/ProxyWebServer.java

```diff
@@ -85,11 +96,19 @@ public void init() throws ServletException {
         super.init();
         getServletContext().setAttribute(ALLUXIO_PROXY_SERVLET_RESOURCE_KEY, proxyProcess);
         getServletContext()
-            .setAttribute(FILE_SYSTEM_SERVLET_RESOURCE_KEY, mFileSystem);
+                .setAttribute(FILE_SYSTEM_SERVLET_RESOURCE_KEY, mFileSystem);
         getServletContext().setAttribute(STREAM_CACHE_SERVLET_RESOURCE_KEY,
-            new StreamCache(Configuration.getMs(PropertyKey.PROXY_STREAM_CACHE_TIMEOUT_MS)));
+                new StreamCache(Configuration.getMs(PropertyKey.PROXY_STREAM_CACHE_TIMEOUT_MS)));
         getServletContext().setAttribute(ALLUXIO_PROXY_AUDIT_LOG_WRITER_KEY, mAsyncAuditLogWriter);
       }
+
+      @Override
+      public void service(final ServletRequest req, final ServletResponse res)
+              throws ServletException, IOException {
+        Stopwatch stopWatch = Stopwatch.createStarted();
+        super.service(req, res);
+        logAccess((HttpServletRequest) req, (HttpServletResponse) res, stopWatch);
```

Comment:
`HttpServletResponse` implements `ServletResponse`, so this isn't guaranteed to be a safe cast? not sure how to get around this as I see you calling methods specific to `HttpServletResponse` besides wrapping the logAccess call around a if instanceof (but i am by far no expert in java)

Reply:
if u look at super.service() it will exception out if ServletResponse is not instanceof HttpServletResponse or ServletRequest is not instanceof HttpServletRequest

Reply:
make it checked to get pass the findbugs build